soh/soh/Enhancements/item-tables/ItemTableTypes.h

```diff
@@ -6,14 +6,22 @@
 #define CHEST_ANIM_SHORT 0
 #define CHEST_ANIM_LONG 1
 
+/**
+ * Flag to indicate which type of Actor has given the player an item. ITEM_FROM_NPC by default,
+ * should be set by the actor giving the item before they send the GetItemEntry signal to the player.
+ */
+typedef enum GetItemFrom {
+    ITEM_FROM_NPC,
+    ITEM_FROM_SKULLTULA,
+    ITEM_FROM_FREESTANDING,
+    ITEM_FROM_CHEST,
+} GetItemFrom;
+
 #define GET_ITEM(itemId, objectId, drawId, textId, field, chestAnim, modIndex, getItemId) \
-    { itemId, field, (chestAnim != CHEST_ANIM_SHORT ? 1 : -1) * (drawId + 1), textId, objectId, modIndex, getItemId, drawId, true, NULL }
+    { itemId, field, (chestAnim != CHEST_ANIM_SHORT ? 1 : -1) * (drawId + 1), textId, objectId, modIndex, getItemId, drawId, true, ITEM_FROM_NPC, NULL }
 
 #define GET_ITEM_NONE \
-    { ITEM_NONE, 0, 0, 0, 0, 0, 0, 0, false, NULL }
-
-#define GET_ITEM_CUSTOM_DRAW(itemId, objectId, drawId, textId, field, chestAnim, modIndex, getItemId, drawFunc) \
-    { itemId, field (chestAnim != CHEST_ANIM_SHORT ? 1 : -1) * (drawId + 1), textId, objectId, modIndex, getItemId, drawId, true, drawFunc }
+    { ITEM_NONE, 0, 0, 0, 0, 0, 0, 0, false, ITEM_FROM_NPC, NULL }
```

Comment:
same question about default

Reply:
For GET_ITEM_NONE it doesn't really matter so I used the same default.